script/verification/Verification.s.sol

```diff
@@ -100,12 +100,14 @@ contract SuperchainRegistry is CommonBase {
         proxies.L1CrossDomainMessenger = stdToml.readAddress(toml, "$.addresses.L1CrossDomainMessengerProxy");
         proxies.L1StandardBridge = stdToml.readAddress(toml, "$.addresses.L1StandardBridgeProxy");
         proxies.SystemConfig = stdToml.readAddress(toml, "$.addresses.SystemConfigProxy");
-        proxies.AnchorStateRegistry = stdToml.readAddress(toml, "$.addresses.AnchorStateRegistryProxy");
-        proxies.DisputeGameFactory = stdToml.readAddress(toml, "$.addresses.DisputeGameFactoryProxy");
+
+        // TODO handle chains which are missing these values:
```

Comment:
Flagging this TODO

Reply:
[3741e88](https://github.com/ethereum-optimism/superchain-ops/pull/418/commits/3741e88f01384df8b2ea69d3533cf25c0abffd2f)